attractions/accordion/accordion-section.svelte

```diff
@@ -1,12 +1,24 @@
 <script>
+  /**
+   * @slot {{ toggle: () => void }} handle
```

Comment:
What about the default slot?

Reply:
They are detected correctly automatically. The only problem is with slots with props.